### PR TITLE
Run3-alca239 Add a file which is used for phi-symmetry calibration using local runs of HCAL

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a file which is used for phi-symmetry calibration using local runs of HCAL. Also. used in several DPG analysis codes. Strange that it was not in the repository for so long

#### PR validation:

The added chi file has been tested externally in the context of phi symmetry studies. Then use the runTheMatrix test workflows to see that it does not interfere with any of the existing workflow.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@tvami You are right. It was tested externally in the context of phi symmetry studies

**Comment by bsunanda** [comment]
@tvami I have edited the main section. Please approve this PR

**Comment by tvami** [comment]
+alca
 * file not use in the main CMSSW, only in private codes
 * on the other hand it would be good to add a wf that actually makes this ALCARECO, and/or consumes it too

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalPedestalLocal_cff.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import HLTrigger.HLTfilters.hltHighLevel_cfi
+hcalminbiasHLT =  HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
+#    HLTPaths = ['HLT_HcalPhiSym'],
+    eventSetupPathsKey='HcalCalMinBias',
+    throw = False #dont throw except on unknown path name 
+)
+
+import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
+hbherecoMBNZS = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
+    digiLabelQIE8  = cms.InputTag("hcalDigiAlCaMB"),
```
Comment (perrotta) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalPedestalLocal_cff.py:19]: Please remove type specifications in this cloned configuration

**Review by perrotta — COMMENTED** [review]
Since you are adding a new file, please make the configurations in it more clean, as suggested

**Comment by perrotta** [comment]
ping @bsunanda

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by tvami** [comment]
+alca
 * the file in this PR is not consumed in central CMSSW, tests are not testing it / wont be affected by it

**Comment by perrotta** [comment]
+1

**Pull request merged**